Makefile

```diff
@@ -51,11 +51,6 @@ test: binaries/client
 
 .PHONY: integration-linux
 integration-linux: binaries/client build-extension-types build-extensions
-# ifdef XDF_CONFIG_HOME
```

Comment:
@Nokel81 probably ok to remove these?

Reply:
Probably, but if a developer has changed settings on a release of Lens then it might fail some integration tests.

Reply:
I don't understand.. these lines were commented out 🤔

Reply:
Oh oops, never mind then. It should be good to delete them.